boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -74,6 +74,16 @@
 	};
 };
 
+arduino_i2c: &i2c0 {
```

Comment:
I think these should not be enabled here.  Shield DTS .overlay files will enable what they need.

(So, I would remove `status = "ok";` from all of these blocks.)

Reply:
For example in K64F DTS: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/frdm_k64f/frdm_k64f.dts#L77